src/app/pages/components/tree/tree.component.ts

```diff
@@ -1,11 +1,19 @@
-import { Component } from '@angular/core';
-
+import { Component, OnInit } from '@angular/core';
+import { ViewChild } from '@angular/core';
+import { TreeComponent as AngularTreeComponent, TREE_ACTIONS, ITreeOptions } from 'angular-tree-component';
+import * as _ from 'lodash';
```

Comment:
no lodash please

Reply:
dont't know how to do that

Reply:
Hey @SalahudinMalik I had replied to [one of the other comments](https://github.com/akveo/ngx-admin/pull/1756#discussion_r204072469) with this to replace your lodash remove :

```typescript
parentNode.data.children = parentNode.data.children.filter((child) => child !== node.data);
```

Can you try it out see if it works?

Reply:
ok

Reply:
just tried its working thanks @dizco